src/config.js

```diff
@@ -241,12 +241,78 @@ const filecoin = {
   addedBy: 'Mostafa Balata',
 };
 
+const viberate = {
+  tokenContract: '0x2c974b2d0ba1716e644c1fc59982a89ddd2ff724',
+  information: {
+    aliasName: 'Viberate',
+    website: 'https://www.viberate.com/',
+    logo: 'https://viberateassets.azureedge.net/favicon.ico',
+  },
+  events: {
+    Mint :{
+      args: {
+        tokens: '_value',
+        sender: '_to',
+      },
+      firstTransactionBlockNumber: 4240691,
+      lastTransactionBlockNumber:  4241086,
+      countTransactions: true,
+      address: '0x2c974b2d0ba1716e644c1fc59982a89ddd2ff724'
+    }
+  },
+  icoParameters: {
+    cap: async(web3, icoContract) => {    
+      const maxCap = await toPromise(icoContract.maxCap)().valueOf();
+      const minCap = await toPromise(icoContract.minCap)().valueOf();
+      return [`Max ${(maxCap / (10 ** 18)).toFixed(2)} VIB`,`Min ${(minCap / (10 ** 18)).toFixed(2)} VIB`];
+    },
+    startDate: async(web3, icoContract) => {      
+      const blockNumber = await toPromise(icoContract.crowdsaleStartBlock)();
+      return (await convertBlockNumberToDate(web3, blockNumber)).formatDate();
+    },
+    endDate: async(web3, icoContract) => {
+      /**
+       * The ICO using `4348935` block number as end date for the crowd sale, 
+       * and this number still is not reached yet, so it will raise an exception that there's 
+       * no timestamp for this block number 
+       */
+      try {
+        const blockNumber = await toPromise(icoContract.crowdsaleEndedBlock)();
+        return (await convertBlockNumberToDate(web3, blockNumber)).formatDate();
+      } catch(e) {
+        return "not reached yet";
+      }
+    },
+    status: async icoContract => 'successful',
+  },
+  matrix: {
+    q1: {answer: true},
+    q2: {answer: true},
+    q3: {answer: true},
+    q4: {answer: true},
+    q5: {answer: true, comment: `Crowdsale contract provides no log event, its provided in the token contract.`},
+    q6: {answer: true},
+    q7: {answer: true},
+    q8: {answer: null},
+    q9: {answer: null},
+    q10: {answer: true},
+    q11: {answer: true},
+    q12: {answer: true},
```

Comment:
question: 'Is price of the token controlled by smart contract?',
@rudolfix Can you check this answer?